.gitignore

```diff
@@ -5,14 +5,15 @@
 # Composer
 composer.phar
 vendor/
+vendor-bin/**/vendor
```

Comment:
when we have `vendor/` in `.gitignore` file, do we really need `vendor-bin/**/vendor`

Reply:
@phil-davis, IMO if we add `/vendor` it will only ignore the top-level directory. But in this case (i.e when using `vendor/`) every `vendor` folder inside the repo is ignored.

Reply:
We need to understand correctly what happens.
@patrickjahns ?

Reply:
I tested this just now. What @paurakhsharma says is true. Every folder called ``vendor`` anywhere in the tree is ignored just by writing ``vendor/`` in ``.gitignore``

@patrickjahns please advise what you want to do. Because we need to have a consistent format for this stuff in every repo. Otherwise it is confusing for people who wonder about small differences.